CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.54.2 / 2024-09-02
+
+* [BUGFIX] Scraping: Fixed significant performance regression in OM Text parsing related to Created Timestamp parsing; that logic has been moved behind an experimental `created-timestamp-zero-ingestion` flag as it should be. #14809
```

Comment:
```suggestion
* [BUGFIX] Scrape: Only parse created timestamps if the related feature-flag (`created-timestamp-zero-ingestion`) is enabled. #14809
```

What do you think? I wouldn't call a performance degradation a bug if it's intentionally behind a feature-flag. I'd call it a bug if we degrade the performance of the usual path without any feature-flag enabled.

Reply:
But I think it's worth documenting [here](https://prometheus.io/docs/prometheus/latest/feature_flags/#created-timestamps-zero-injection) that it's expected poor performance when OM text 1.0 is the used protocol